### PR TITLE
Case activity api to aat/add flux config

### PR DESCRIPTION
**Pull request by MisterSK**
### JIRA link (if applicable) ###
[AB#796](https://dev.azure.com/hmcts/90472b63-dc8d-4ae6-8819-a32bd211e914/_workitems/edit/796)

### Change description ###
Add AAT flux config for ccd-case-activity-api

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -57,6 +57,8 @@ k8s/**/common/ccd/user-profile-api.yaml @hmcts/cdm
 k8s/**/common/ccd/api-gateway-web.yaml @hmcts/cdm
 k8s/aat/common/ccd/case-print-service.yaml @hmcts/cdm
 k8s/aat/common/ccd/test-stubs-service.yaml @hmcts/cdm
+k8s/aat/common/ccd/api-gateway-web.yaml @hmcts/cdm
```
Comment (timja) [inline CODEOWNERS]: why is this here?

File: CODEOWNERS [hunk 2]
```diff
@@ -57,6 +57,8 @@ k8s/**/common/ccd/user-profile-api.yaml @hmcts/cdm
...
+k8s/**/common/ccd/case-activity-api.yaml @hmcts/cdm
```
Comment (timja) [inline CODEOWNERS]: please deploy aat and prod separately

Comment (MisterSK) [reply CODEOWNERS]: Hi Tim, realised this. Correcting...

**Comment by MisterSK** [comment]
Closing this PR to add new AAT-only PR to isolate prod / aat deployments.
